src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -46,18 +48,20 @@ export class OverflowMenuPane extends Dialog {
 				return position.addOffset(pos, -20, -60);
 			}
 			return position.addOffset(pos, -20, 60);
-		}
+		};
 
 		if (!this.dialogConfig.menuLabel) {
 			this.dialogConfig.menuLabel = this.i18n.get().OVERFLOW_MENU.OVERFLOW;
 		}
 
-		setTimeout(() => this.listItems()[0].focus());
+		setTimeout(() => {
+			getFocusElementList(this.elementRef.nativeElement).every(button => button.tabIndex = -1);
```

Comment:
It's not checking if `tabIndex` is already set so it will override user set values.
```suggestion
			getFocusElementList(this.elementRef.nativeElement).every(button => {
				if (button.tabIndex === undefined) {
					button.tabIndex = -1;
				}
			});
```
should do the trick but test it.